THIRD-PARTY-PROGRAMS

```diff
@@ -610,227 +610,3 @@ THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 IN THE SOFTWARE.
-
-10. LLVM (src/graph/backend/graph_compiler/core/src/util/array_ref.hpp)
-==============================================================================
-The LLVM Project is under the Apache License v2.0 with LLVM Exceptions:
-==============================================================================
```

Comment:
Hi @vgvozdeva , please note that the TPP file is changed here. It may affect the BOM file as well. Thanks.